templates/index.html

```diff
@@ -18,7 +18,7 @@
     <div class="container">
         <div class="route-index">
             {{#each this}}
-            <a href="?path={{this.path}}#{{this.method}}">
+            <a href="?path={{encodeUri this.path}}#{{encodeUri this.method}}">
```

Comment:
I don't think you need to encode the method.

Reply:
Good catch.  Thanks for working with me on this!